src/ble/ble_watch_service.c

```diff
@@ -188,8 +188,13 @@ static uint32_t tx_char_add(ble_watchs_t * p_watchs)
 
     memset(&attr_md, 0, sizeof(attr_md));
 
+#if defined(INSECURE_PAIRING) && INSECURE_PAIRING == 1
```

Comment:
@Emeryth something like this: when you normally compile this will default to secure pairing... but if you build like `INSECURE_PAIRING=1 make` it will use mode_set_open